gnosis/cowswap
# Added linter step with auto-fix

# Summary

Adding linter to CI with auto-fix.


  # To Test

~Will test by adding an intentional change without proper linting~

It applied `prettier` fixes to a couple of files, check the changes.

:warning: Terms and conditions and Cookie policy have quite a lot of fixes :warning: 
They can be ignored if the new way is not desirable.
Anyway, as far as I've checked the contents is the same
cc @avsavsavs 

  # Background

Discussed during the team sync as a way to help people without the dev setup to make contributions to the code/content
